nnstreamer.pc.in

```diff
@@ -4,6 +4,7 @@ prefix=@PREFIX@
 exec_prefix=@EXEC_PREFIX@
 libdir=@LIB_INSTALL_DIR@
 includedir=@INCLUDE_INSTALL_DIR@
+test_templatedir=@TEST_TEMPLATE_DIR@ # test template dir to be used
```

Comment:
this is .pc file, not .ini file.

Reply:
This variable is meant to locate where the templates are in a standard way, so it is inside `.pc`

This directory can be accessed as:
```sh
$ TEMPLATE_DIR=$(pkg-config nnstreamer --variable=test_templatedir)
$ echo $TEMPLATE_DIR # result varies depending on the platform
/usr/lib/nnstreamer/bin/unittest-nnstreamer
$ ls $TEMPLATE_DIR
nnstreamer-test.ini.in  subplugin_unittest_template.cc.in
```

or in meson
```meson
nnst_dep = dependency('nnstreamer',)
nnst_dep.get_pkgconfig_variable('test_templatedir') 
nnst_dep.get_variable(pkgconfig: 'test_templatedir') ## from meson > 0.51
```